### DISCUSSION TITLE
Access SD Card ?

### DISCUSSION THREAD

**Discussion by HalbesHaehnchen** [phil-barrett/grblHAL-teensy-4.x#95] [Q&A] [answered]
Hello again. I got my machine up and running, auto squaring works, thanks for the help again. 
Now I am curious about the sd card slot of the teensy. Is there a way to access stored gcode files ? This way I could turn off the computer for longer jobs.

**Comment by phil-barrett** [comment, marked as answer]
Yes, You need to build with the SD card plug in. After you install the firmware, in ioSender, an SD Card tab will show up.  Make sure a card is plugged in and upload gcode files.  The tab should show a directory of the files.  Right click a file for run options.  

![image](https://github.com/phil-barrett/grblHAL-teensy-4.x/assets/16570958/50ea18a7-941b-4c0c-8dbb-850372f28d4e)

**Reply by HalbesHaehnchen** [reply]
Thanks again sir. Just downloaded iosender because I found it handy for setting things up. Didn't know it can also access the sd card.

**Reply by phil-barrett** [reply]
While you are at it, check out the probe tab.
